frontend/packages/core/src/text.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import { Fab, Grid } from "@material-ui/core";
+
+import { ClipboardButton } from "./button";
+
+const CopyButtonContainer = styled(Grid)({
+  marginLeft: "7px",
+});
+
+const Pre = styled.pre({
+  border: "1px solid rgba(13, 16, 48, 0.38)",
+  backgroundColor: "rgba(13,16,48,0.12)",
+  borderRadius: "4px",
+  fontSize: "16px",
+  color: "#242b8c",
```

Comment:
colors are a bit off to me actually (bg too dark, blue text is odd), did you not like the figma choices? i forget.

Reply:
Yeah I thought it looked slightly better with more contrast than the designs Kanchan had